sdk/src/client.rs

```diff
@@ -70,6 +70,17 @@ impl NetworkClient {
             .expect("SP1_VERIFIER_ADDRESS must be 20 bytes")
     }
 
+    pub async fn get_nonce(&self) -> u64 {
+        let res = self
+            .rpc
+            .get_nonce(GetNonceRequest {
+                address: self.auth.get_address().to_vec(),
+            })
+            .await
+            .unwrap();
+        res.nonce
+    }
```

Comment:
docs: Add a more clear specification for what a nonce is and where you're querying it from? i.e. querying from sequencer

Reply:
added